src/components/result/result.css

```diff
@@ -0,0 +1,30 @@
+:root {
```

Comment:
I don't think that we need to do it in every component. Maybe App.css will be enough?

Reply:
I just have it before merge due to have a real design to ckeck. At final stage we can sure have only one :)